process-migration/image.yaml

```diff
@@ -0,0 +1,64 @@
+schema_version: 1
+
+name: "rhpam-7/rhpam75-process-migration-openshift"
+description: "Red Hat Process Automation Manager Process Migration 7.5 container image"
+version: "1.0"
+from: "ubi8:8-released"
+labels:
+    - name: "com.redhat.component"
+      value: "rhpam-7-rhpam75-process-migration-openshift-rhel8-container"
+    - name: "io.k8s.description"
+      value: "Platform for running Red Hat Process Automation Manager Process Migration"
+    - name: "io.k8s.display-name"
+      value: "Red Hat Process Automation Manager Process Migration 7.5"
+    - name: "io.openshift.expose-services"
+      value: "8080:http"
+    - name: "io.openshift.tags"
+      value: "javaee,rhpam,rhpam7"
+envs:
+    - name: "SCRIPT_DEBUG"
+      example: "true"
+      description: "If set to true, ensures that the bash scripts are executed with the -x option, printing the commands and their arguments as they are executed."
+    - name: JBOSS_KIE_EXTRA_CLASSPATH
+      example: "mariadb-java-client-2.4.2.jar,postgresql-42.2.6.jar"
```

Comment:
How would these jars make it into the image?

In KIE Server, we do it like this for mariadb and postgresql: https://github.com/jboss-container-images/jboss-kie-modules/tree/master/jboss-kie-db-drivers

For other drivers, we use extension images: https://github.com/jboss-container-images/rhpam-7-openshift-image/tree/master/templates/contrib/jdbc

One could also use other solutions, like custom s2i image builds to add the drivers, or via mounting a persistent volume with the drivers there. What solution were you envisioning?

Reply:
In Thorntail is very simple, you just need to add the driver anywhere you want and add it to the EXTRA_CLASSPATH env var that will add it to the app classpath (thanks to the script).
It would be better to add it in an immutable way for simplicity. That means, creating a new image from the original one.

Reply:
@ruromero I think it's a bit of a pain for the user to have to create their own child image when we have other mechanisms in use today for other purposes (as mentioned above), but I get it. I think this is fine for now, but perhaps an area for improvement in the future if we want to make it easier on the user.